### PR TITLE
add some more category of product

### PR DESCRIPTION
**Pull request by Rockyraven**
some category of product change 
live link:   https://quickstore-react.netlify.app/

### CODE REVIEW

File: src/assets/index.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+import sport5 from './sport5.jpg'
+import sport3 from './sports3.jpeg'
+import sport4 from './sports4.jpg'
+import sneaker1 from './sneaker.jpeg'
+import sneaker2 from './sneaker2.jpeg'
 
-
-export {brandBanner, menShoe, womenShoe, kidShoe, imgShoe1, imgShoe2, imgShoe3, imgShoe4, imgShoe5, imgShoe6 }
\ No newline at end of file
+export {brandBanner, casual1, casual2, casual3, formal1, formal2, formal3, sneaker1, sneaker2,  sport2, sport3, sport4, sport5}
```
Comment (Shubhamkr2610) [inline src/assets/index.js]: Appreciation: good of importing and exporting images

**Review by Shubhamkr2610 — APPROVED** [review]
your code is clean and readable 
every thing looks good to me , you are good to go

File: src/assets/index.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-import menShoe from "./mens-shoe.jpg"
-import womenShoe from './women-shoe.jpg'
-import kidShoe from './kidshoe.jpg'
-import imgShoe1 from './shoe.jpg'
-import imgShoe2 from './shoe1.jpeg'
-import imgShoe3 from './shoe2.jpeg'
-import imgShoe4 from './shoe3.jpeg'
-import imgShoe5 from './shoe4.jpeg'
-import imgShoe6 from './shoe5.jpeg'
+import casual1 from './casual.webp'
```
Comment (arjun-computer-geek) [inline src/assets/index.js:2]: This is good way of using image

**Review by arjun-computer-geek — APPROVED** [review]
Looks good. Clean code.

**Pull request merged**